runtime-kernel/linux-kernel-rpi64-lts/autobuild/prepare

```diff
@@ -0,0 +1,8 @@
+unset CFLAGS
+unset CPPFLAGS
+unset LDFLAGS
+
+ln -sv /usr/bin/ld.bfd "$SRCDIR"/ld
+export PATH="$SRCDIR:$PATH"
+
+sed -e 's|^EXTRAVERSION = -rc.*|EXTRAVERSION =|g' -i Makefile
```

Comment:
```suggestion
abinfo "Setting PKGDIR to the parent directory ..."
export PKGDIR="$(realpath $SRCDIR/../abdist)"
```


You can remove this `sed` - we are not going to build -rc kernels anyway, and as @MingcongBai stated, you can set `PKGDIR` to the parent of the kernel source tree to avoid compiled modules being deleted by `make mrproper`.

![image](https://github.com/user-attachments/assets/7bf73886-37f3-4aaf-834c-633ce341cd4c)